src/.github/workflows/publish.yml

```diff
@@ -11,7 +11,7 @@ concurrency:
 
 jobs:
   publish-docker-images:
-    if: github.event_name == 'push' && contains(github.ref, 'refs/tags/')
+    if: contains(github.ref, 'refs/tags/')
```

Comment:
Hmm, did this worked to generate nightly image in case of a push on main?

Reply:
I just realized I removed this line in my pull request https://github.com/okp4/actions/pull/9

Reply:
Totally right ! I removed it as the event could only be a `push`, but the entire condition should be removed to allow nightly image publication.